uboot-kirkwood: fix goflexhome/net bootcommand

bobafetthotmail opened the pull request:
Goflexhome/net use uImage, and to boot an uImage the u-boot
must use bootm command, not bootz.

Fixes the "i cannot boot LEDE with this u-boot" issue that I
found out myself with my goflexnet.

Signed-off-by: Alberto Bursi <alberto.bursi@outlook.it>

@blogic Can this patch be sent also in 17.01 branch?

bobafetthotmail commented:
@stintel can this patch be sent also in 17.01?

mkresin commented:
Please improve your commit message. You know how it should look like. "bootz -> bootm" isn't really descriptive.

You are changing more than just switching from zImage to uImage. You are touching the mtdparts definition and your are switching from loading kernel from nand to loading kernel from an ubi volume. As long as it isn't mentioned in the commit it is consider as mistake.

bobafetthotmail commented:
Updated commit description.

> You are changing more than just switching from zImage to uImage. You are touching the mtdparts definition and your are switching from loading kernel from nand to loading kernel from an ubi volume. As long as it isn't mentioned in the commit it is consider as mistake.

No, I'm just modifying a patch that does that and github gets confused, see the file in master https://github.com/lede-project/source/blob/master/package/boot/uboot-kirkwood/patches/150-goflexhome.patch

EDIT: it seems that after my update github isn't confused anymore and shows only the line I changed.

nbd168 commented:
Merged to my staging tree, thanks.

stintel commented:
@bobafetthotmail please don't mention random people. I don't know anything about uboot.
